site/resources/views/livewire/finder.blade.php

```diff
@@ -0,0 +1,266 @@
+                {{ trans('courses.finder.selected_cards') }}
+            </span>
+            <div class="dropdown d-inline-block" @click.stop>
+                <button
+                    class="btn border-0"
+                    type="button"
+                    data-bs-toggle="dropdown"
+                    aria-expanded="false"
+                    @click.stop="openMenu($el, true)"
+                >
+                    <i class="fa-solid fa-ellipsis-vertical"></i>
+                </button>
+                <ul class="dropdown-menu">
+                    <li
+                        class="dropdown-item d-flex cursor-pointer align-items-center d-flex d-sm-none"
+                        @click="selectAll"
+                    >
+                        <span class="flex-fill me-5">
+                            {{ trans('courses.finder.select_all')}}
+                        </span>
+                    </li>
+                    <li class="d-block d-sm-none">
+                        <hr class="dropdown-divider">
+                    </li>
+                    @can('massActionsForCardAndFolder', $course)
+                    <li
+                        class="dropdown-item d-flex cursor-pointer align-items-center"
+                        data-bs-toggle="modal"
+                        data-bs-target='{{"#$modalMoveId"}}'
+                        :data-bs-keys="selectedItems"
+                        dusk="multi-movein-option"
+                    >
+                        <i class="fa-solid fa-arrow-right-to-bracket me-2"></i>
+                        <span class="flex-fill me-5">
+                            {{ trans('courses.finder.move_in')}}
+                        </span>
+                    </li>
+                    <li
+                        class="dropdown-item d-flex cursor-pointer align-items-center"
+                        wire:click="cloneMultiple(selectedItems)"
+                        dusk="multi-copy-option"
+                    >
+                        <i class="fa-solid fa-clone me-2"></i>
+                        <span class="flex-fill me-5">
+                            {{ trans('courses.finder.menu.copy')}}
+                        </span>
+                    </li>
+                    <li
+                        class="dropdown-item d-flex cursor-pointer align-items-center"
+                        data-bs-toggle="modal"
+                        data-bs-target='{{"#$modalCloneId"}}'
+                        :data-bs-keys="selectedItems"
+                        dusk="multi-clonein-option"
+                    >
+                        <i class="fa-solid fa-file-import me-2"></i>
+                        <span class="flex-fill me-5">
+                            {{ trans('courses.finder.clone_in')}}
+                        </span>
+                    </li>
+                    <li
+                        wire:confirm="{{ trans('courses.finder.menu.delete.card.confirm') }}"
```

Comment:
Custom directive "confirm" not needed anymore with Livewire v3.0.8. Update Livewire & remove js starting at https://github.com/unil-lettres/impact/blob/development/site/resources/views/layouts/app-js.blade.php#L31 and below.

Reply:
Updated Livewire to 3.1.0. Done.